tests/test_sheet_collector.py

```diff
@@ -395,3 +395,34 @@ def test_sheet_collector_collect_files_prints_output(tmpdir, test_data, capfd):
     captured = capfd.readouterr()
     for print_item in test_data["collect_files_test"]["expected_print"]:
         assert print_item in captured.out
+
+def test_empty_cells_in_sheets_get_type_none():
+    """Check that blank cells are type None."""
+    
+    from sheetshuttle import util
+
+    # create SheetCollector object
+    #   -pass in key and directory of group's config
+    my_sheet_collector = sheet_collector.SheetCollector(
+        key_file="key3.json", sources_dir="config/sheet_sources"
+        )
+    # create api variable
+    api = my_sheet_collector.sheets
+
+    # expected data with None in blank cells
+    expected_data = [
+    ["col1", "col2", "col3", "col4"],
+    ["fizz", None, "fooz", "fizz2"],
+    ["buzz", "fooz2", "buzz2", None],
+    ]
+
+    # execute an API call to get data from google sheets
+    data_from_call = sheet_collector.Sheet.execute_sheets_call(
+        api, "1b1e0ml8klRF4VOq03D1M_Quk_33KBYONigQ3WXhbNkU", "sheet1", "A1", "D3"
+        )
+
+    # append None where there is a mismatch in data and number of columns/rows
+    new_data = util.fill_to_dimensions(data_from_call, 4, 3)
+
+    # check to see if empty elements get None
+    assert new_data == expected_data
```

Comment:
Let's try to avoid tests that rely on using the Google Sheets API. The credentials you're using here and not accessible in github actions and the test case will fail on all machines that don't have your credentials. Instead, let's only test the function that you changed, aka `fill_to_dimensions()`. I appreciate you trying to check that the whole process works as expected but this will complicate things for GitHub actions. Let's keep this simple.